core/src/test/java/nav/statusplattform/core/repositories/HelpTextRepositoryTest.java

```diff
@@ -21,6 +25,17 @@ public class HelpTextRepositoryTest {
     private final HelpTextRepository helpTextRepository = new HelpTextRepository(dbContext);
     private DbContextConnection connection;
 
+    private ArrayList<Integer> numbers = new ArrayList<Integer>(Arrays.asList(1, 2, 3, 4, 5));
```

Comment:
The diff does not show where else this is used besides `getRandomizedHelpTextEntity()`.

If that is:
- the case, it should not be a class-wide member
- not the case, it needs a more descriptive name and/or documentation